styles.css

```diff
@@ -89,16 +89,53 @@ article {
     width: 80%;
 }
 
+main {
+    overflow: auto;
+}
+
 a {
     text-decoration: none;
     color: #F77913;
 }
 
+.recipeContainer {
+    width: 50%;
+    padding: 10px;
+    float: left;
+}
+
+.recipe {
+    width: 100%;
+    overflow: auto;
+    background-color: #FFFFFF;
+    border: 1px solid #F77913;
+    border-radius: 5px;
+    padding: 10px;
+    float: left;
+}
+
+.recipe div {
+    float: left;
+    padding: 0 5px;
+}
+
+.recipe img {
```

Comment:
You have a class 'recipeimg' that isn't being used. Also, the image size is never specified anywhere. We should set it because we don't know if we'll receive different size images from the API. I'd suggest 80x80px

Reply:
Also, if following the suggestion for divs above, add a margin to the right and bottom of the image of 10px each.